services/blockchain-app-registry/tests/functional/shared/utils/downloadRepository.test.js

```diff
@@ -72,8 +72,8 @@ describe('Test getFileDownloadURL method', () => {
 	it('should return correct file download info when file is valid', async () => {
 		/* eslint-disable-next-line no-useless-escape */
 		const fileUrlRegex = new RegExp(`^https:\/\/\\w*.github.com\/repos\/LiskHQ\/${config.gitHub.appRegistryRepoName}\/contents\/devnet\/Enevti\/nativetokens.json\\?owner=LiskHQ&repo=${config.gitHub.appRegistryRepoName}&ref=${config.gitHub.branch}$`);
```

Comment:
There's no need for a regular expression here. It can be a simple string expression.
And, `expect(response).toMatch(fileUrlRegex);` can simply be replaced by `expect(response).toBe(expectedFileURL);`.

Reply:
Discussed offline.